webrtc/videoroom/config/runtime.exs

```diff
@@ -17,8 +16,8 @@ get_env = fn env, default ->
   end
 end
 
-host = get_env.("HOST", "localhost")
-port = get_env.("PORT", default_port) |> String.to_integer()
+host = get_env.("VIRTUAL_HOST", "localhost")
+port = 4000
```

Comment:
If we change the port to fixed `4000` we should probably update `README.md` to reflect that.

Reply:
good point!